aclk/aclk.c

```diff
@@ -408,7 +408,7 @@ void aclk_graceful_disconnect(mqtt_wss_client client)
         }
     }
     netdata_log_info("ACLK link is down");
```

Comment:
any reason this remains `netdata_log_info`

Reply:
I have not replaced everything. Old log lines are still valid, they are mapped to the new logging system.